package.json

```diff
@@ -231,6 +231,22 @@
           "markdownDescription": "Insert space before end of self closing tag. \nExample:\n  ```<tag/> -> <tag />```. Default is `true`. See [here](https://github.com/redhat-developer/vscode-xml/wiki/Formatting#xmlformatspacebeforeemptyclosetag-) for more information.",
           "scope": "window"
         },
+        "xml.format.xsiSchemaLocationSplit": {
+          "type": "string",
+          "enum": [
+            "onElement",
+            "onPair",
+            "none"
+          ],
+          "markdownEnumDescriptions": [
+            "Split `xsi:schemaLocation` content after each namespace declaration",
+            "Split `xsi:schemaLocation` content after each location declaration",
```

Comment:
Maybe adjust these descriptions to be a bit more specific:
* "Put a new line after each namespace and URI, arranging them into one column"
* "Put a new line after each URI, arranging the content into pairs of namespace and URI"

Reply:
fixed